submissions/DaniaB24/js-dom/js/main.js

```diff
@@ -0,0 +1,50 @@
+const container = document.querySelector(".content-block");
+const listOfCreatures = document.querySelector("#list-of-creatures");
+
+let dotaHeroes = [
```

Comment:
use `let `just in case where you change value of variable. in every another case use `const`